4-appfactory/apps/cymbal-bank/main.tf

```
@@ -15,6 +15,7 @@
  */
 
 locals {
+  services_with_infra = ["userservice", "ledgerwriter"]
```

You can add a comment here:

```suggestion
  # Services in this list will receive dedicated projects for application-specific infrastructure, such as an app-specific database.
  services_with_infra = ["userservice", "ledgerwriter"]
```